features/gba-dev/waardetabel-gba.feature

```diff
@@ -0,0 +1,247 @@
+      | naam.adellijkeTitelPredicaat.soort        | <soort>        |
+
+      Voorbeelden:
+      | relatie   | relatiedan | waarde | omschrijving | soort     | fields   |
+      | 'kind'    | kind       | BS     | barones      | titel     | kinderen |
+      | ouder '1' | ouder      | JH     | jonkheer     | predicaat | ouders   |
+      | 'partner' | partner    | R      | ridder       | titel     | partners |
+
+    Rule: wanneer voor de code geen bijbehorende waarde voorkomt in de tabel, wordt alleen de code geleverd
+
+      Abstract Scenario: code voor <element> komt niet voor in de tabel <tabel>
+        Gegeven de persoon met burgerservicenummer '000000231' heeft de volgende gegevens
+        | <element> |
+        | <waarde>  |
+        Als gba personen wordt gezocht met de volgende parameters
+        | naam                | waarde                          |
+        | type                | RaadpleegMetBurgerservicenummer |
+        | burgerservicenummer | 000000231                       |
+        | fields              | <groep>.<veld>                  |
+        Dan heeft de response een persoon met alleen de volgende '<groep>' gegevens
+        | naam                | waarde         |
+        | <veld>.code         | <waarde>       |
+
+        Voorbeelden:
+        | element                           | tabel               | groep                        | veld                    | waarde |
+        | geboorteplaats (03.20)            | Gemeenten           | geboorte                     | plaats                  | 9876   |
+        | geboorteland (03.30)              | Landen              | geboorte                     | land                    | 1234   |
+
+      Abstract Scenario: code voor <element> komt niet voor in de tabel <tabel>
+        Gegeven de persoon met burgerservicenummer '000000243' heeft de volgende '<groep>' gegevens
+        | <element> |
+        | <waarde>  |
+        Als gba personen wordt gezocht met de volgende parameters
+        | naam                | waarde                          |
+        | type                | RaadpleegMetBurgerservicenummer |
+        | burgerservicenummer | 000000243                       |
+        | fields              | <groep>                         |
+        Dan heeft de response een persoon met alleen de volgende '<groep>' gegevens
+        | naam                | waarde         |
+        | <veld>.code         | <waarde>       |
+
+        Voorbeelden:
+        | element                           | tabel               | groep          | veld                    | waarde |
+        | plaats overlijden (08.20)         | Gemeenten           | overlijden     | plaats                  | 9876   |
+        | land overlijden (08.30)           | Landen              | overlijden     | land                    | 1234   |
+        | aanduiding verblijfstitel (39.10) | Verblijfstiteltabel | verblijfstitel | aanduiding              | 01     |
```

Comment:
Heet de tabel echt 'Verblijfstiteltabel' of toch gewoon 'Verblijfstitel'?
Nu wordt de titel van het scenario 

`code voor aanduiding verblijfstitel (39.10) komt niet voor in de tabel Verblijfstiteltabel`

wat een beetje vreemd is.

Reply:
Goed gezien. Aangepast